packages/frames.js/src/middleware/farcaster.ts

```diff
@@ -14,7 +14,7 @@ function isValidFrameActionPayload(
 ): value is FrameActionPayload {
   return (
     typeof value === "object" &&
-    !!value &&
+    value !== null &&
```

Comment:
would this not incorrectly pass for `value = undefined`? Perhaps Boolean(value) is better?

Reply:
`value = undefined` will not pass through `typeof value === 'object'`.